sqlanalytics/resource_sql_endpoint.go

```diff
@@ -186,6 +186,8 @@ func ResourceSQLEndpoint() *schema.Resource {
 	s := common.StructToSchema(SQLEndpoint{}, func(
 		m map[string]*schema.Schema) map[string]*schema.Schema {
 		m["auto_stop_mins"].Default = 120
+		m["auto_stop_mins"].Required = false
+		m["auto_stop_mins"].Optional = true
```

Comment:
`.Optional = true` and no `omitempty` creates a schema consistency conflict.

Reply:
that's why it has `.Required = false`